### DISCUSSION TITLE
How to fix this?

### DISCUSSION THREAD

**Discussion by Wallyy07** [coder/code-server#3239] [Q&A] [answered]
![IMG_20210427_210643](https://user-images.githubusercontent.com/83290044/116252503-36bf9d80-a7a2-11eb-9868-8a8c8ae299bb.jpg)

When I run "code-server" its says lib/node not found

**Comment by jsjoeio** [comment]
Hmm..do you have `node` installed on your machine? You can test by running `node --version`

If not, download here: https://nodejs.org/en/download/

**Reply by Wallyy07** [reply]
Nice thanks!

**Comment by jsjoeio** [comment, marked as answer]
I would suggest [reading the docs](https://github.com/cdr/code-server/blob/main/docs/install.md) on how to install code-server and trying again.
